RecoMuon/TrackingTools/plugins/MuonErrorMatrixAdjuster.h

```diff
@@ -95,11 +95,12 @@ class MuonErrorMatrixAdjuster : public edm::EDProducer {
   bool theRescale;
 
   /// holds the error matrix parametrization
-  edm::ParameterSet theMatrixProvider_pset;
-  MuonErrorMatrix* theMatrixProvider;
+  std::unique_ptr<MuonErrorMatrix> theMatrixProvider;
 
   /// hold on to the magnetic field
   edm::ESHandle<MagneticField> theField;
```

Comment:
Could this be `const`?

Reply:
The class MuonErrorMatrix has no `const` methods and therefore making it `const` would not work:
https://github.com/cms-sw/cmssw/blob/master/RecoMuon/TrackingTools/interface/MuonErrorMatrix.h

Reply:
Hi @Dr15Jones , actually I was referring to `edm::ESHandle<MagneticField> theField`

Reply:
ah, thanks for the explanation